src/ImageRunModal.jsx

```diff
@@ -68,13 +68,37 @@ const PublishPort = (props) => {
             </div>
         </React.Fragment>
     );
-};
+
+const EnvVar = (props) =>
```

Comment:
Same here, please decompose props so that it's easy to see at a glance what this expects.